ddr/lib/ddr-scanner/pdb/PdbScanner.cpp

```diff
@@ -223,30 +223,30 @@ PdbScanner::loadDataFromPdb(const wchar_t *filename, IDiaDataSource **dataSource
 		}
 
 		if (DDR_RC_OK != rc) {
-			ERRMSG("Creating instance of IDiaDataSource failed with HRESULT = %08X\n", hr);
+			ERRMSG("Creating instance of IDiaDataSource failed with HRESULT = %08lX\n", hr);
```

Comment:
Is this correct for both 32 and 64 bit Windows compiles?  Just verifying that we didn't switch one warning for another.